packages/payment-detection/src/near-detector.ts

```diff
@@ -58,28 +63,29 @@ export default class NearNativeTokenPaymentDetector extends ReferenceBasedDetect
   };
 
   /**
-   * Extracts the events for an address and a payment reference
+   * Extracts payment events from an and a payment reference
    *
-   * @private
-   * @param address Payment address
-   * @param eventName Is it for payment or refund
-   * @param network The payment network name
+   * @param address Address to check
+   * @param eventName Indicate if it is an address for payment or refund
+   * @param requestCurrency The request currency
    * @param paymentReference The reference to identify the payment
-   * @param paymentNetworkVersion
-   * @returns The balance with events
+   * @param paymentNetwork the payment network
+   * @returns The balance
```

Comment:
Either call it `extractBalance` or leave the comment, I don't understand this mix.